lsp-perl.el

```diff
@@ -50,6 +50,29 @@
   :group 'lsp-perl
   :package-version '(lsp-mode . "6.3"))
 
+(defcustom lsp-perl-perlCmd nil
+  "Path to perl interpreter used in Perl Language Server.  Defaults to `perl' if nil."
+  :type 'string
+  :group 'lsp-perl)
+(defcustom lsp-perl-perlInc nil
+  "A vector of paths to add to perl library path."
+  :type '(repeat string)
```

Comment:
this should be lsp-vector-string. Note that emacs 27 native json parsing requires vector, not a list. FIx here and in the review.